Woof/WoofTests/Model/KeyValueStorage/Extensions/KeyValueStorage+UnitTests.swift

```diff
@@ -13,4 +18,11 @@ extension KeyValueStorage {
             self.deleteData(for: $0.rawValue)
         }
     }
+
+    /**
+     Removes the associated data for `userPreferences` key.
+     */
+    func removeAll() {
+        deleteData(for: KeyValueStorage.Key.userPreferences)
+    }
```

Comment:
Confusing name for the method, because I can call it for any KeyValueStorage and don't get what `remove all ` means.